lib/cc/cli/engines/install.rb

```diff
@@ -3,6 +3,8 @@
 module CC
   module CLI
     module Engines
+      include CC::Analyzer
```

Comment:
Ah nice catch. `CC::Analyzer` is already included in the `EngineCommand` class so `Install` has access. Not sure why I added `Analyzer` here.